### PR TITLE
Shares report

### PR DESCRIPTION
**Pull request by ijunaid8088**
/cc @azharmalik3

### CODE REVIEW

File: app/assets/stylesheets/admin/admin.scss [hunk 1]
```diff
@@ -540,7 +540,7 @@ ul.columns {
   margin-right: 20px;
 }
 
-#users_datatables_processing, #cameras_datatables_processing {
+#users_datatables_processing, #cameras_datatables_processing, #camera_shares_datatables_processing {
```
Comment (ijunaid8088) [inline app/assets/stylesheets/admin/admin.scss]: Sorry.. These ids are auto-generated by DataTables, they are going to be in this way.

File: app/assets/javascripts/admin/camera_shares.js.coffee [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+  camera_shares_datatables.init
+    src: $('#camera_shares_datatables')
+    onSuccess: (grid) ->
+      # execute some code after table records loaded
+      return
+    onError: (grid) ->
+      # execute some code on network or other general error
+      return
+    onDataLoad: (grid) ->
```
Comment (azharmalik3) [inline app/assets/javascripts/admin/camera_shares.js.coffee]: remove empty function.

Comment (ijunaid8088) [inline app/assets/javascripts/admin/camera_shares.js.coffee]: Its done!

File: app/assets/javascripts/admin/camera_shares.js.coffee [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+appendMe = ->
+  $("#div-dropdown-checklist").css({"visibility": "visible", "width": "20px", "top": "1px", "float": "right", "right": "22px" })
+  $(".dataTables_info").css("display", "none")
+  $(".dataTables_length > label").css("display", "none")
+  $("#camera_shares_datatables_paginate > .pagination-panel").css("display", "none")
+  $(".paging_bootstrap_extended").css("float","none")
+
+linkCamera = (name, type, row) ->
+  console.log name
```
Comment (azharmalik3) [inline app/assets/javascripts/admin/camera_shares.js.coffee]: Remove `console.log`

**Pull request merged**